src/sqlfluff/rules/L016.py

```diff
@@ -1,8 +1,13 @@
 """Implementation of Rule L016."""
 
-from typing import Optional, Tuple
```

Comment:
I added type hints to this file. Most of the changes are related to that, not the actual fix.